packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -109,6 +109,12 @@ exports.sourceNodes = ({ actions, schema }) => {
       fix
       enhancement
     }
+
+    type LimitsYaml implements Node @dontInfer {
+      id: ID!
+      name: String!
+      value: String!
```

Comment:
This is not the schema proposed in the issue (separate numeric and text keys there so they can be combined and we can apply number formatting as part of the UI implementation)

Reply:
> This is not the schema proposed in the issue (separate numeric and text keys there so they can be combined and we can apply number formatting as part of the UI implementation)

I guess this is related to the previous comment so I'll just wait for your response on that one.